project.clj

```diff
@@ -76,14 +79,16 @@
                  [postgresql "9.3-1102.jdbc41"]                       ; Postgres driver
                  [io.crate/crate-jdbc "2.1.6"]                        ; Crate JDBC driver
                  [prismatic/schema "1.1.5"]                           ; Data schema declaration and validation library
+                 [redux "0.1.4"]                                      ; Utility functions for building and composing transducers
                  [ring/ring-core "1.6.0"]
                  [ring/ring-jetty-adapter "1.6.0"]                    ; Ring adapter using Jetty webserver (used to run a Ring server for unit tests)
                  [ring/ring-json "0.4.0"]                             ; Ring middleware for reading/writing JSON automatically
                  [stencil "0.5.0"]                                    ; Mustache templates for Clojure
+                 [tide "0.1.0-SNAPSHOT"]                              ; Various algorithms for working with timeseries
                  [toucan "1.0.3"                                      ; Model layer, hydration, and DB utilities
                   :exclusions [honeysql]]]
   :repositories [["bintray" "https://dl.bintray.com/crate/crate"]]    ; Repo for Crate JDBC driver
-  :plugins [[lein-environ "1.1.0"]                                    ; easy access to environment variables
+  :plugins [[lein-environ "1.1.0"]                                    ; easy access to environment variables            
```

Comment:
Extra whitespace. I guess the linter doesn't check this file. I highly recommend you add

```emacs-lisp
(add-hook 'before-save-hook #'delete-trailing-whitespace)
```

to your `init.el` so it will automatically clean up these sorts of things for you